### PR TITLE
[#2175] Display time of mouse position and not current time in tooltip.

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @cadecairos received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/5bc7f0d86868421/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @cadecairos received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/8c1b031724e80b1/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/5bc7f0d86868421/output.txt

Total script time: 2.16 mins

Server is running on http://pr966.butyr.org/templates/basic/
Run the Test Suite: http://pr966.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/8c1b031724e80b1/output.txt

Total script time: 3.00 mins

File: css/editors.less [hunk 1]
```diff
@@ -464,7 +464,7 @@
   &:hover > .butter-tooltip {
     opacity: 1;
     visibility: visible;
-    .transition( all 0.3s ease 0.5s );
+    .transition( opacity 0.3s ease 0.5s );
```
Comment (cadecairos) [inline css/editors.less]: While this transition time is adequate for most other tool tips, I feel it is a tad bit too laggy for this application. Would it be simple enough to override this style with a faster transition to opaque?

That is, if you agree it'd be a good change.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @cadecairos received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/30b008120ae7b85/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @cadecairos received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/6a67c9b0e6db742/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/30b008120ae7b85/output.txt

Total script time: 1.27 mins

Server is running on http://pr966.butyr.org/templates/basic/
Run the Test Suite: http://pr966.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/6a67c9b0e6db742/output.txt

Total script time: 1.67 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/20f698c248b54a4/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/3ca949d12172a34/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/20f698c248b54a4/output.txt

Total script time: 1.87 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/3ca949d12172a34/output.txt

Total script time: 1.55 mins

Server is running on http://pr966.butyr.org/templates/basic/
Run the Test Suite: http://pr966.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @cadecairos received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/ee27cef8fbd52ce/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @cadecairos received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/061170dce088957/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/ee27cef8fbd52ce/output.txt

Total script time: 1.20 mins

Server is running on http://pr966.butyr.org/templates/basic/
Run the Test Suite: http://pr966.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/061170dce088957/output.txt

Total script time: 2.09 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @cadecairos received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/d3f699044ae9e5f/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @cadecairos received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/c1c3057fff18498/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/d3f699044ae9e5f/output.txt

Total script time: 1.72 mins

Server is running on http://pr966.butyr.org/templates/basic/
Run the Test Suite: http://pr966.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/c1c3057fff18498/output.txt

Total script time: 3.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/a5a5d139aa5745d/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/00eab29ef6718c4/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/a5a5d139aa5745d/output.txt

Total script time: 1.66 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/00eab29ef6718c4/output.txt

Total script time: 0.87 mins

Server is running on http://pr966.butyr.org/templates/basic/
Run the Test Suite: http://pr966.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/59d830b118fdac6/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/59d830b118fdac6/output.txt

Total script time: 0.62 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/67ba22a763c4b93/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/25cc6257fdd4b06/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/67ba22a763c4b93/output.txt

Total script time: 3.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/25cc6257fdd4b06/output.txt

Total script time: 1.63 mins

Server is running on http://pr966.butyr.org/templates/basic/
Run the Test Suite: http://pr966.butyr.org/test/

File: css/tray-status-bar.less [hunk 2]
```diff
@@ -5,6 +5,7 @@
 @_mediaStatusContainerHeight: @_butterStatusHeight;
 @_playButtonHeight: 26px;
 @_scrubberBarHeight: 8px;
+@_scrubberBarHeight: 8px;
```
Comment (mjschranz) [inline css/tray-status-bar.less]: Can't believe I didn't notice this but you duplicated a variable here! Surprised lint doesn't complain.

**Pull request merged**